common/include/PS2Edefs.h

```diff
@@ -564,6 +564,8 @@ typedef void (CALLBACK* _PS2EsetEmuVersion)(const char* emuId, u32 version);		//
 // GS
 // NOTE: GSreadFIFOX/GSwriteCSR functions CANNOT use XMM/MMX regs
 // If you want to use them, need to save and restore current ones
+typedef void (CALLBACK* _GSosdLog)(const char *utf8, u32 color);
```

Comment:
I guess monitor stay on screen. Would it be possible to fuse the 2 callbacks and to add a flag option of the behavior. This way, it can still be extended later with new flag. What do you think about it ?